app/server/api/transfer_card_api.py

```diff
@@ -13,7 +13,12 @@ class TransferCardAPI(MethodView):
     @requires_auth
     def get(self):
 
-        transfer_cards = TransferCard.query.all()
+        only_loaded = request.args.get('only_active', 'true').lower() == 'true'
+
+        if only_loaded:
+            transfer_cards = TransferCard.query.filter(TransferCard.amount_loaded_signature != None).all()
+        else:
+            transfer_cards = TransferCard.query.all()
```

Comment:
This issue wasn't introduced in this PR, but should we switch this to `paginate_query` at some point?

Reply:
Yeah I thought about this too, but I think given the way transfer cards are loaded onto phones is quite different (there's potentially location based sharding etc, it made more sense to do as a seperate PR)